### PR TITLE
Localization

### PR DESCRIPTION
**Pull request by royzumbado**
### Update wording

#182 

### Steps to test

1.Open EOSRate in spanish
2. It should show all text now without Lorep Ipsum and properly translated in Spanish

### Checklist
- [ ] I have added/updated tests to cover these changes.
- [ ] The branch name, commit history and PR title follow [conventions](https://developers.eoscostarica.io/docs/open-source-guidelines#pull-request-general-guidelines).
- [ ] I have taken into consideration any impact to site performance.

### CODE REVIEW

File: services/frontend/src/language/es.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
     "settingsTitle": "Configuración",
     "settingsLanguages": "Ingles/Español",
-    "settingsNotifications": "Notificación",
+    "settingsNotif  ications": "Notificación",
```
Comment (arturocr) [inline services/frontend/src/language/es.json]: Please remove the whitespace in `settingsNotifications`

Comment (royzumbado) [reply services/frontend/src/language/es.json]: Removed and sent a commit to _localization branch in my fork. Please review and let me know if there´s anything else I´m missing, thank you.

image

Comment (arturocr) [reply services/frontend/src/language/es.json]: @royzumbado I'm still seeing the space in the [file](https://github.com/royzumbado/eos-rate/blob/_localization/services/frontend/src/language/es.json), did you push your commit?

Comment (royzumbado) [reply services/frontend/src/language/es.json]: @arturocr oh, I missed the push, I have pushed it now via GitHub desktop, could you please review/confirm?

Comment (arturocr) [reply services/frontend/src/language/es.json]: Looks good. 😃

**Pull request merged**